lib/std/elf.zig

```diff
@@ -1730,6 +1730,14 @@ pub const SHN_COMMON = 0xfff2;
 /// End of reserved indices
 pub const SHN_HIRESERVE = 0xffff;
 
+// Legal values for ch_type (compression algorithm). used in Elf{32,64}_Chdr
+pub const ELFCOMPRESS_ZLIB = 1;
+pub const ELFCOMPRESS_ZSTD = 2;
+pub const ELFCOMPRESS_LOOS = 0x60000000;
+pub const ELFCOMPRESS_HIOS = 0x6fffffff;
+pub const ELFCOMPRESS_LOPROC = 0x70000000;
+pub const ELFCOMPRESS_HIPROC = 0x7fffffff;
```

Comment:
Mind the fully qualified namespace - you're already in `std.elf`:

```suggestion
pub const COMPRESS_ZLIB = 1;
pub const COMPRESS_ZSTD = 2;
pub const COMPRESS_LOOS = 0x60000000;
pub const COMPRESS_HIOS = 0x6fffffff;
pub const COMPRESS_LOPROC = 0x70000000;
pub const COMPRESS_HIPROC = 0x7fffffff;
```

Better yet:

```suggestion
pub const COMPRESS = struct {
    pub const ZLIB = 1;
    pub const ZSTD = 2;
    pub const LOOS = 0x60000000;
    pub const HIOS = 0x6fffffff;
    pub const LOPROC = 0x70000000;
    pub const HIPROC = 0x7fffffff;
};
```

Still, we can do even better:

```suggestion
pub const COMPRESS = enum {
    ZLIB = 1,
    ZSTD = 2,
    LOOS = 0x60000000,
    HIOS = 0x6fffffff,
    LOPROC = 0x70000000,
    HIPROC = 0x7fffffff,
    _,
};
```